tiledb/sm/fragment/fragment_info.cc

```diff
@@ -149,29 +144,29 @@ Status FragmentInfo::get_sparse(uint32_t fid, int32_t* sparse) const {
     return LOG_STATUS(Status_FragmentInfoError(
         "Cannot check if fragment is sparse; Sparse argument cannot be null"));
 
-  if (fid >= fragments_.size())
+  if (fid >= fragment_num())
     return LOG_STATUS(Status_FragmentInfoError(
         "Cannot check if fragment is sparse; Invalid fragment index"));
 
-  *sparse = (int32_t)fragments_[fid].sparse();
+  *sparse = (int32_t)single_fragment_info_vec_[fid].sparse();
 
   return Status::Ok();
 }
 
 uint32_t FragmentInfo::fragment_num() const {
-  return (uint32_t)fragments_.size();
+  return (uint32_t)single_fragment_info_vec_.size();
 }
 
 Status FragmentInfo::get_cell_num(uint32_t fid, uint64_t* cell_num) const {
```

Comment:
Move to new coding standard?

Reply:
Let me do this in a separate pass everywhere. This is an old function, I usually move to the new standard new functions or functions I significantly refactor in my pass.